rfc/rfc003-oauth2-authorization.md

```diff
@@ -235,12 +235,12 @@ The **vcs** field contains the authorization credentials that determine the acce
 
```

Comment:
`The **vcs** field contains the authorization credentials.`
This must be updated so it explains that it can contain multiple credentials?
Also, currently we only have a authorization credential, so perhaps explain more types can be added in the future and each type will have its own set of rules how en when they should be used?

Reply:
Fixed